docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,520 @@
+
+</Admonition>
+
+## Teleport instances
+
+Run the Teleport Auth Service and Proxy Service as a scalable group of compute
+resources, for example, a Kubernetes `Deployment` or AWS Auto Scaling group.
+This requires running the `teleport` binary on each Kubernetes pod or virtual
+machine or in your group. In the [Configuration](#configuration) section, we
+will show you how to configure each binary for high availability.
+
+### Open ports
+
+Ensure that, on each Teleport instance, the following ports allow traffic from
+the load balancer. The Proxy Service uses these ports to communicate with
+Teleport users and services.
+
+As with your load balancer configuration, the ports you should open on your
+Teleport instances depend on whether you will enable TLS Routing: 
+
+<Tabs>
+<TabItem label="TLS Routing">
+
+| Port | Description |
+| - | - |
+| `443` | ALPN port for TLS Routing. |
+
+</TabItem>
+<TabItem label="Separate Ports">
+
+These ports are required:
+
+| Port | Description |
+| - | - |
+| `3023` | SSH port for clients connect to. |
+| `3024` | SSH port used to create reverse SSH tunnels from behind-firewall environments. |
+| `443` | HTTPS connections to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. |
+
+You can leave these ports closed if you are not using their corresponding
+services:
+
+| Port | Description |
+| - | - |
+| `3026` | HTTPS Kubernetes proxy |
+| `3036` | MySQL port |
+| `5432` | Postgres port |
+
+</TabItem>
+</Tabs>
+
+*This is the same table of ports you used to configure the load balancer.*
+
+### License file
+
+If you are deploying Teleport Enterprise, you need to download a license file
+and make it available to your Teleport Auth Service instances.
+
+To obtain your license file, visit the [Teleport customer
+dashboard](https://dashboard.gravitational.com/web/login) and log in. Click
+"DOWNLOAD LICENSE KEY". You will see your current Teleport Enterprise account
+permissions and the option to download your license file:
+
+![License File modal](../../img/enterprise/license.png)
+
+The license file must be available to each Teleport Auth Service instance at
+`/var/lib/teleport/license.pem`. 
+
+### Configuration
+
+Create a configuration file and provide it to each of your Teleport instances at
+`/etc/teleport.yaml`. We will explain the required configuration fields for a
+high-availability Teleport deployment below. These are the minimum requirements,
+and when planning your high-availability deployment, you will want to follow a
+more specific [deployment guide](introduction.mdx) for your environment. 
+
+#### `storage` 
+
+The first configuration section to write is the `storage` section, which
+configures the key-value store and object store for the Teleport Auth Service.
+
+The example configurations below apply to Teleport deployments that use Amazon
+DynamoDB, Google Cloud Firestore, and etcd as the Auth Service backend:
+
+<Tabs>
+<TabItem label="Amazon DynamoDB">
```

Comment:
A lot of this is covered in more detail in the backends doc page. Should we be trying to maintain a single source of truth instead of duplicating that info here?